packages/keychain/src/utils/connection/index.ts

```diff
@@ -12,22 +12,27 @@ import { deployFactory } from "./deploy";
 import { openSettingsFactory } from "./settings";
 import { normalize } from "@cartridge/utils";
 import { switchChain } from "./switchChain";
+import { AuthOption, AuthOptions } from "@cartridge/controller";
 
 export function connectToController<ParentMethods extends object>({
   setRpcUrl,
   setContext,
   setController,
+  setConfigSignupOptions,
 }: {
   setRpcUrl: (url: string) => void;
   setContext: (ctx: ConnectionCtx) => void;
   setController: (controller?: Controller) => void;
+  setConfigSignupOptions: (options: AuthOption[] | undefined) => void;
 }) {
   return connectToParent<ParentMethods>({
     methods: {
       connect: normalize(
         connect({
           setRpcUrl,
           setContext,
+          setConfigSignupOptions: (options: AuthOptions | undefined) =>
```

Comment:
Small nit - can the method signature above be `options?: AuthOptions`? So we can just pass the name here